browser-tests/snapshot-tests/snapshot.spec.js

```diff
@@ -0,0 +1,104 @@
+const { test, expect } = require("@playwright/test");
+const path = require("path");
+const fs = require("fs");
+
+test("Snapshots", async ({ page }) => {
+
+  test.setTimeout(120000)
+  for (const pageName of Object.keys(pagesToTest)) {
+    const contexts = pagesToTest[pageName];
+    const contextsToTest = contexts.length > 0 ? contexts : [ undefined ];
```

Comment:
Could this potentially always append `undefined` so we don't need to add it to the pagesToTest contexts?

```
const contextsToTest = [...contexts, undefined];
```

Reply:
There are pages that are only ever displayed with contexts so we don't want `undefined` on them